backend/src/app/rpc/commands/binfile.clj

```diff
@@ -960,12 +961,13 @@
   "Import a penpot file in a binary format."
   {::doc/added "1.15"
    ::webhooks/event? true}
-  [{:keys [::db/pool] :as cfg} {:keys [::rpc/profile-id project-id file] :as params}]
+  [{:keys [::db/pool] :as cfg} {:keys [::rpc/profile-id project-id file version] :as params}]
```

Comment:
I don't think we need to pass version as parameter, you can access the constant directly on runtime on the JVM accessing to the app.common.files.defaults